### DISCUSSION TITLE
[@types/node-fetch] misses FormData as a typing or I am wrong.

### DISCUSSION THREAD

**Discussion by Arsabutispik** [DefinitelyTyped/DefinitelyTyped#66499] [Issues with a @types package]
Before updating to the latest version I had no problem with it but now it says FormData is not exported and that I add the export myself

[a comment by an automated account is not shown]

**Comment by nikcorg** [comment]
@arsabutispik that's right. `FormData` is not a fetch (or node-fetch) specific thing and thus isn't exported by the package's type definition.

Use the [FormData definition from the DOM types](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.formdata.html) provided by TypeScript.

**Reply by Arsabutispik** [reply]
Alright thank you, it wasn't a problem before so that's why I asked
